CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Migrated to SDK API 35.0 (firmware 0.89.0).
 - `flipperzero_test::tests` now allows `#[cfg(..)]` attributes on test methods.
 
+### Removed
```

Comment:
Oops, I missed that this section was added to a previous release. I'll fix it in my next PR.